test/extra_node_modules/improve-global/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  improve: '@apostrophecms/global',
+  options: {
+    testGlobalLevelLoaded: true,
```

Comment:
One flag just to say this even loaded, and another to try to change a value which we may or may not override as a test.